python/core/auto_generated/geometry/qgsgeometryutils.sip.in

```diff
@@ -410,12 +410,21 @@ angular deviation (in radians) allowed when testing for regular point spacing.
 .. versionadded:: 3.14
 %End
 
-    static int segmentSide( const QgsPoint &pt1, const QgsPoint &pt3, const QgsPoint &pt2 ) /HoldGIL/;
+    static int segmentSide( const QgsPoint &linePoint1, const QgsPoint &linePoint2, const QgsPoint &point ) /HoldGIL/;
```

Comment:
This is an api break, as it would break any python code which uses named keywords, e.g.

    side = QgsGeometryUtils.segmentSide(pt1=..., pt2=..., pt3=...)

Reply:
Oh no, I wasn't aware..
The background was that the documentation was simply wrong mixing up pt3 and pt2. And as the point names were so confusing that even the code author seemed to have confused them I suggested more meaningful names for easier use.

What are the options now? 
- Wait for a later QGIS version that implements several API changes and pull it later there 
- Keep the api and correct only the documentation
- ...?

I don't have any rush but I can also change the code to the less meaningful names. What's your suggestion/favorite?

Reply:
> Keep the api and correct only the documentation

That's really the only option. An api breaking release isn't anywhere on the horizon right now.